pkg/odo/cli/catalog/list/list.go

```diff
@@ -13,18 +13,16 @@ const RecommendedCommandName = "list"
 // NewCmdCatalogList implements the odo catalog list command
 func NewCmdCatalogList(name, fullName string) *cobra.Command {
 	components := NewCmdCatalogListComponents(componentsRecommendedCommandName, util.GetFullName(fullName, componentsRecommendedCommandName))
-	services := NewCmdCatalogListServices(servicesRecommendedCommandName, util.GetFullName(fullName, servicesRecommendedCommandName))
 
 	catalogListCmd := &cobra.Command{
 		Use:     name,
-		Short:   "List all available component & service types.",
-		Long:    "List all available component and service types from OpenShift",
-		Example: fmt.Sprintf("%s\n\n%s\n", components.Example, services.Example),
+		Short:   "List all available component types.",
+		Long:    "List all available component types from OpenShift",
```

Comment:
`from OpenShift` is not correct anymore. That was the case when we were still doing s2i.